ansible/templates/settings.py

```diff
@@ -44,6 +44,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'rest_framework',
+    'widget_tweaks',
```

Comment:
This requirement should not be necessary on the workers? Does not hurt to have it, though, as it is installed anyway.

Reply:
I guess you're right, if those changes add up, it might be better to have seperate templates for master/worker nodes.